core/cache/charmconfigwatcher.go

```diff
@@ -234,14 +235,18 @@ func (w *CharmConfigWatcher) setConfigHash() (bool, error) {
 		}
 	}
 
+	hashCache, _ := newHashCache(
```

Comment:
There's an old typo of mine in the comment for this method "checkConfig" needs changing to "setConfigHash" :)